pkg/ui/workspaces/cluster-ui/src/databaseDetailsV2/tablesView.tsx

```diff
@@ -8,216 +8,252 @@
+  [
+    {
+      title: TableColName.NAME,
+      width: "15%",
+      sorter: true,
+      render: (t: TableRow) => {
+        // This linking is just temporary. We'll need to update it to the correct path
+        // using db ID and table ID once we have the table details page.
+        const encodedDBPath = EncodeDatabaseTableUri(t.dbName, t.name);
+        return <Link to={encodedDBPath}>{t.qualifiedNameWithSchema}</Link>;
+      },
+      sortKey: TableSortOption.NAME,
     },
-  },
-  {
-    title: TableColName.REPLICATION_SIZE,
-    width: "fit-content",
-    sorter: true,
-    render: (t: TableRow) => {
-      return Bytes(t.replicationSizeBytes);
+    {
+      title: TableColName.REPLICATION_SIZE,
+      width: "fit-content",
+      sorter: true,
+      render: (t: TableRow) => {
+        return Bytes(t.replicationSizeBytes);
+      },
+      sortKey: TableSortOption.REPLICATION_SIZE,
     },
-  },
-  {
-    title: TableColName.RANGE_COUNT,
-    width: "fit-content",
-    sorter: true,
-    render: (t: TableRow) => {
-      return t.rangeCount;
+    {
+      title: TableColName.RANGE_COUNT,
+      width: "fit-content",
+      sorter: true,
+      render: (t: TableRow) => {
+        return t.rangeCount;
+      },
+      sortKey: TableSortOption.RANGES,
     },
-  },
-  {
-    title: TableColName.COLUMN_COUNT,
-    width: "fit-content",
-    sorter: true,
-    render: (t: TableRow) => {
-      return t.columnCount;
+    {
+      title: TableColName.COLUMN_COUNT,
+      width: "fit-content",
+      sorter: true,
+      render: (t: TableRow) => {
+        return t.columnCount;
+      },
+      sortKey: TableSortOption.COLUMNS,
     },
-  },
-  {
-    title: TableColName.NODE_REGIONS,
-    width: "20%",
-    render: (t: TableRow) => (
-      <div>
-        {Object.entries(t.nodesByRegion ?? {}).map(([region, nodes]) => (
-          <RegionNodesLabel
-            key={region}
-            nodes={nodes}
-            region={{ label: region, code: region }}
-          />
-        ))}
-      </div>
-    ),
-  },
-  {
-    title: TableColName.LIVE_DATA_PERCENTAGE,
-    sorter: true,
-    render: (t: TableRow) => {
-      return (
+    {
+      title: TableColName.NODE_REGIONS,
+      width: "20%",
+      render: (t: TableRow) => (
         <div>
-          <div>{t.liveDataPercentage * 100}%</div>
+          {Object.entries(t.nodesByRegion ?? {}).map(([region, nodes]) => (
+            <RegionNodesLabel
+              key={region}
+              nodes={nodes}
+              region={{ label: region, code: region }}
+            />
+          ))}
+        </div>
+      ),
+    },
+    {
+      title: TableColName.LIVE_DATA_PERCENTAGE,
+      sorter: true,
+      width: "fit-content",
+      sortKey: TableSortOption.LIVE_DATA,
+      render: (t: TableRow) => {
+        return (
           <div>
-            {Bytes(t.liveDataBytes)} / {Bytes(t.totalDataBytes)}
+            <div>{t.liveDataPercentage * 100}%</div>
```

Comment:
nit: We should maybe talk to Inna about what level of precision we want and should round to that. Also one thing about rounding logic that we should keep in mind is that we shouldnt round "99.99x" up to 100%